### DISCUSSION TITLE
please give some examples how to access fields in json object, thanks

### DISCUSSION THREAD

**Discussion by workhours** [nlohmann/json#3400] [Q&A] [answered]
#### Please describe the steps to reproduce the issue.

1.
2.
3.

#### Which compiler and operating system are you using?

- Compiler: ___
- Operating system: ___

#### Which version of the library did you use?

- [ ] latest release version 3.10.5
- [ ] other release - please state the version: ___
- [ ] the `develop` branch

#### If you experience a compilation error: can you [compile and run the unit tests](https://github.com/nlohmann/json#execute-unit-tests)?

- [ ] yes
- [ ] no - please copy/paste the error message below

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/features/element_access/.
